### DISCUSSION TITLE
nx migrate includes which packages?

### DISCUSSION THREAD

**Discussion by bjornsgithub** [nrwl/nx#4917] [General]
I've checked the documentation and it's not clear to me which packages are included using nx migrate and which are not? I'm trying to combine nx with an automated dependency update tool renovate and can I expect nx migrate handles all the dependencies defined within the package.json?

**Comment by ph55** [comment]
Not clear for me ass well.

If I want, for example, to update `@angular/cli` package, should I wait for `nx migrate` to include it or I can simply run `ng update` ?

**Comment by dan2kx** [comment]
I've just updated to today's beta version of nx so that i can test out next 11 and it gave a warning about the "typescript" minimum version being 4.3.5 (iirc), which got me wondering... can i update typescript? which modules will nx migrate actually take care of? i couldn't find anything detailing this
